app/drivers/adc.c

```diff
@@ -3,11 +3,9 @@
 #include "drivers/gpio.h"
 #include "drivers/adc.h"
 
-#define NUM_OF_CHANNELS 2
-
 ADC_HandleTypeDef adc_handle;
 static bool m_initialized;
-static uint16_t m_adc_values[NUM_OF_CHANNELS];
+static uint16_t m_adc_values[ADC_NUM_OF_CHANNELS];
```

Comment:
I feel like this should be NUM_OF_ADC_CHANNELS to match the other enum NUM_* stuff

Reply:
Just a bit worried about name clashes when there isn’t a prefix

Reply:
Just a bit worried about name clashes when there isn’t a prefix

Reply:
Just a bit worried about name clashes when there isn’t a prefix

Reply:
Just a bit worried about name clashes when there isn’t a prefix